pkg/controller/servicemesh/controlplane/htpasswd.go

```diff
@@ -32,9 +32,13 @@ func (r *controlPlaneInstanceReconciler) patchHtpasswdSecret(ctx context.Context
 			log.Error(err, "failed to generate the HTPasswd password")
 			return err
 		}
-		h := sha1.New()
-		h.Write([]byte(rawPassword))
-		auth = "internal:{SHA}" + base64.StdEncoding.EncodeToString(h.Sum(nil))
+		hashedPassword, err1 := bcrypt.GenerateFromPassword([]byte(rawPassword), bcrypt.DefaultCost)
+		if err1 != nil {
+			log.Error(err1, "failed to encrypt the raw password")
+			return err1
+		}
+		username := "internal"
```

Comment:
Great job introducing this variable. Helps explain what "internal" is. In the old code, it wasn't clear at all.